packages/flutter_tools/lib/src/compile.dart

```diff
@@ -124,10 +128,11 @@ Future<String> compile(
 /// restarts the Flutter app.
 class ResidentCompiler {
   ResidentCompiler(this._sdkRoot, {bool trackWidgetCreation: false,
-      String packagesPath})
+      String packagesPath, CompilerMessageConsumer compilerMessageConsumer: printError})
     : assert(_sdkRoot != null),
       _trackWidgetCreation = trackWidgetCreation,
-      _packagesPath = packagesPath {
+      _packagesPath = packagesPath,
+      stdoutHandler = new _StdoutHandler(consumer: compilerMessageConsumer) {
```

Comment:
Would it make sense to make the same change above on line 111 ?

Reply:
Maybe in the future.